package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tezster",
-  "version": "0.2.1",
+  "version": "0.2.2-beta",
```

Comment:
version change should always happen on dev or master. 
For this CL it's fine. Next time, you need to be mindful about this.